### PR TITLE
Add session cache tests for Infinispan in a container

### PR DESCRIPTION
**Pull request by mswatosh**
Session cache test modified to run with Infinispan in a container using testcontainers.

### CODE REVIEW

File: dev/com.ibm.ws.session.cache_fat_infinispan_container/fat/src/com/ibm/ws/session/cache/fat/infinispan/container/SessionCacheOneServerTest.java [hunk 1]
```diff
@@ -37,23 +39,15 @@
 
     public static SessionCacheApp app = null;
 
-    // TODO this is temporarily set to single-threaded in order to ensure coverage of codepath without actually enabling concurrent use yet (only invokeAll is used)
-    public static final ExecutorService executor = Executors.newFixedThreadPool(1); // TODO Executors.newFixedThreadPool(12);
+    public static final ExecutorService executor = Executors.newFixedThreadPool(12);
```
Comment (njr-11) [inline dev/com.ibm.ws.session.cache_fat_infinispan_container/fat/src/com/ibm/ws/session/cache/fat/infinispan/container/SessionCacheOneServerTest.java]: Should we wait on enabling concurrent testing as we do elsewhere?

**Comment by mswatosh** [comment]
#build

**Comment by mswatosh** [comment]
#build

**Comment by mswatosh** [comment]
#build

**Comment by mswatosh** [comment]
#build

**Comment by mswatosh** [comment]
The one failing test has a matching defect.

**Pull request merged**